app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -41,10 +41,10 @@ import org.oppia.util.parser.ExplorationHtmlParserEntityType
 import java.util.Date
 import javax.inject.Inject
 
-const val STATE_FRAGMENT_PROFILE_ID_ARGUMENT_KEY = "STATE_FRAGMENT_PROFILE_ID_ARGUMENT_KEY"
-const val STATE_FRAGMENT_TOPIC_ID_ARGUMENT_KEY = "STATE_FRAGMENT_TOPIC_ID_ARGUMENT_KEY"
-const val STATE_FRAGMENT_STORY_ID_ARGUMENT_KEY = "STATE_FRAGMENT_STORY_ID_ARGUMENT_KEY"
-const val STATE_FRAGMENT_EXPLORATION_ID_ARGUMENT_KEY = "STATE_FRAGMENT_EXPLORATION_ID_ARGUMENT_KEY"
+const val STATE_FRAGMENT_PROFILE_ID_ARGUMENT_KEY = "StateFragmentPresenter.profile_id"
+const val STATE_FRAGMENT_TOPIC_ID_ARGUMENT_KEY = "StateFragmentPresenter.topic_id"
+const val STATE_FRAGMENT_STORY_ID_ARGUMENT_KEY = "StateFragmentPresenter.story_id"
+const val STATE_FRAGMENT_EXPLORATION_ID_ARGUMENT_KEY = "StateFragmentPresenter.exploration_id"
 private const val TAG_AUDIO_FRAGMENT = "AUDIO_FRAGMENT"
```

Comment:
TAG should be changed too!!
as per the pr title says about Tags

Reply:
@rt4914 please confirm here, as tags are not mentioned in the description.

Reply:
Tags related to fragment / activities should be mentioned in their respective companion objects.

Example: `internal const val TAG = "AudioFragment.tag"`

Reply:
done